Objective-C/CBLReplicator.mm

```diff
@@ -551,13 +556,18 @@ - (void) onDocsEnded: (NSArray<CBLReplicatedDocument*>*)docs pushing: (BOOL)push
             continue;
         
         if (!pushing && c4err.domain == LiteCoreDomain && c4err.code == kC4ErrorConflict) {
-            // Conflict pulling a document -- the revision was added but app needs to resolve it:
-            CBLLogInfo(Sync, @"%@: pulled conflicting version of '%@'", self, doc.id);
-            NSError* error;
-            if ([_config.database resolveConflictInDocument: doc.id error: &error]) {
-                [doc resetError]; // Reset the error as successfully resolving the conflicts
-            } else
-                CBLWarn(Sync, @"%@: Conflict resolution of '%@' failed: %@", self, doc.id, error);
+            dispatch_async(_otherDispatchQueue, ^{
+                // Conflict pulling a document -- the revision was added but app needs to resolve it
+                CBLLogInfo(Sync, @"%@: pulled conflicting version of '%@'", self, doc.id);
+                NSError* error;
+                if ([_config.database resolveConflictInDocument: doc.id
```

Comment:
Make sure to catch exception here. If you plan to do it in a separate commit, please add TODO: here.

Reply:
added TODO: